docs/guides.md

```diff
@@ -544,67 +544,176 @@ secureboot-certs
 ```
 
 :::tip
-Running `secureboot-certs` only needs to be done once per host, not per VM.  It must also be called after ISO upgrades once per upgraded host.
+Running `secureboot-certs` only needs to be done once per host, not per VM. It is not pool-wide. It must also be called after ISO upgrades once per upgraded host.
 :::
 
-### Configure the Guest VM
+### Enable Secure Boot for a Guest VM
 
-#### Setup for Linux VMs
+#### Creating a new Secure Boot VM
+
+This section describes how to create a new Secure Boot VM using the various XCP-ng interfaces.
+
+:::warning
+If the VM will be running Linux, ensure that the distribution supports installation with Secure Boot enabled prior to enabling Secure Boot for the VM on XCP-ng.  If the distribution does not support installation with Secure Boot enabled, then first create the VM with Secure Boot disabled and enable it *only after following the distribution's instructions for enabling Secure Boot*.
+:::
+
+##### Create a Secure Boot VM using XO
+
+During VM creation in XO, go to the *Advanced* section and select **uefi** as the **Boot firmware**.  This will display a **Secure boot** toggle that can be clicked to enable Secure Boot.
+
+![](../assets/img/screenshots/xo_uefi_sb_create_option.png)
+
+##### Create a Secure Boot VM using XCP-ng Center
+
+When creating a VM in XCP-ng Center, select the firmware option  *UEFI with SecureBoot* to enable Secure Boot.
+
+##### Create a Secure Boot VM using `xe`
+
+If creating a VM using `xe`, select UEFI firmware using the command:
+
+```
+# Set the VM to use UEFI firmware
+xe vm-param-set uuid=<vm-uuid> HVM-boot-params:firmware=uefi
+```
+
+... and then enable Secure Boot using the command:
+
+```
+# Enable Secure Boot for the VM
+xe vm-param-set uuid=<vm-uuid> platform:secureboot=true
+```
```

Comment:
I don't think many users create VMs with `xe`. It's one of the tasks that is difficult from CLI because you first need to select a template, add your parameters, disks, etc. So I think we can avoid this section since it's covered by the "Enabling Secure Boot for an Existing UEFI VM" section.

Reply:
Sounds good.

Reply:
Done